test/db/cmd/cmd_print

```diff
@@ -431,7 +431,7 @@ EXPECT=<<EOF
   },
   {
     "offset": 5,
-    "esil": "rcx,rdx,==,$z,zf,:=,64,$b,cf,:=,$p,pf,:=,63,$s,sf,:=,63,$o,of,:=",
+    "esil": "rcx,rdx,==,$z,zf,:=,64,$b,cf,:=,$p,pf,:=,63,$s,sf,:=,rcx,9223372036854775808,-,!,63,$o,^,of,:=,3,$b,af,:=",
```

Comment:
uh this 9223372036854775808 is 0x8000000000000000.. couldnt be represented in a better way? esil supports hex values too in numbers.